addon/ng2/blueprints/ng2/files/karma.conf.js

```diff
@@ -2,10 +2,16 @@ module.exports = function(config) {
   config.set({
     basePath: '',
     frameworks: ['jasmine'],
+    plugins: [
+      require('karma-jasmine'),
+      require('karma-chrome-launcher')
```

Comment:
let's include it then. Can you rebase the PR against the current master changes?